[DOCS] screenshot/video limitation (#3657)
elastic/observability-docs: docs/en/observability/synthetics-create-test.asciidoc

* https://playwright.dev/docs/api/class-video[Videos]
* The https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-screenshot-1[`toHaveScreenshot`] and https://playwright.dev/docs/api/class-snapshotassertions[`toMatchSnapshot`] assertions

[NOTE]
====

> Captures done programmatically via https://playwright.dev/docs/api/class-page#page-screenshot[`screenshot`] or https://playwright.dev/docs/api/class-page#page-video[`video`] are not stored nor can be shown in the Synthetics application. Providing a `path` will likely make the monitor fail due to missing permissions to write local files.

Reviewer: ```suggestion
Captures done programmatically via https://playwright.dev/docs/api/class-page#page-screenshot[`screenshot`] or https://playwright.dev/docs/api/class-page#page-video[`video`] are not stored and are not shown in the Synthetics application. Providing a `path` will likely make the monitor fail due to missing permissions to write local files.
```
